src/expr.cpp

```diff
@@ -795,8 +797,13 @@ ExprParser::Token ExprParser::Lex(std::string *error) {
 }
 
 ExprParser::Token ExprParser::PopOperand(std::string *error) {
-    Token t = Token::From();
-    if(stack.empty() || stack.back().type != TokenType::OPERAND) {
+    Token t;
```

Comment:
Don't think so, I think I was just explicitly calling constructor for readability. Should I revert this change?

Reply:
i FIXED IT NOW In case of error a "proper" `TokenType::ERROR` with `*expr = NULL` is returned. It probably does not matter but this was the old behaviour.